### PR TITLE
Localized replacement of `trusty` repo only to Mesosphere packages.

### PR DESCRIPTION
**Pull request by asridharan**

### CODE REVIEW

File: mesos/Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ push: check-version
 mesos: check-version
 	sed "s/VERSION/$(VERSION)/g" dockerfile-templates/$@ > $@
 	if test `echo $(VERSION) | grep rc`; then \
-	  sed -i -e "s/trusty/trusty-testing/g" $@ ; \
+	  sed -i -e "s/{MESOS-TRUSTY}/trusty-testing/g" $@ ; \
```
Comment (karya0) [inline mesos/Makefile]: I am wondering if we should rename this to `MESOSPHERE_REPO` instead and then use `sed` to replace it with `trusty` or `trusty-testing`. Any thoughts?

Comment (asridharan) [inline mesos/Makefile]: That's better. Will make the change.

File: mesos/Makefile [hunk 2]
```diff
@@ -29,7 +29,7 @@ push: check-version
 mesos: check-version
 	sed "s/VERSION/$(VERSION)/g" dockerfile-templates/$@ > $@
 	if test `echo $(VERSION) | grep rc`; then \
-	  sed -i -e "s/trusty/trusty-testing/g" $@ ; \
+	  sed -i -e "s/{MESOSPHERE_REPO}/trusty-testing/g" $@ ; \
```
Comment (karya0) [inline mesos/Makefile]: If it's not RC, then we need to replace `{MESOSPHERE_REPO}` with `trusty` :)

**Comment by karya0** [comment]
Merged.

**Pull request closed without merging**